.github/workflows/lint.yml

```diff
@@ -0,0 +1,68 @@
+name: Static analysis
+on:
+  pull_request:
+  push:
+    branches:
+      - trunk
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
+
+      - name: Setup Node.js
+        uses: Automattic/vip-actions/nodejs-setup@trunk
+        with:
+          node-version-file: .nvmrc
```

Comment:
Can you use the VIP action for this? Less boilerplate and implements caching that saves time and $$

https://github.com/Automattic/vip-actions/blob/trunk/nodejs-setup/README.md

Reply:
I don't think you need to setup PHP for this job btw

Reply:
Good point, I had run into some problems and so I had re-added the checkout code. It can be removed since the vip-action already takes care of that. I'll take out the PHP code as well

Reply:
Done